pkg/tf2pulumi/convert/tf.go

```diff
@@ -207,6 +207,16 @@ func getTrivia(sources map[string][]byte, r hcl.Range) (hclwrite.Tokens, hclwrit
 		}
 	}
 
+	if len(tokens) > 0 && first == last {
```

Comment:
I'll have a quick look into this, feels like this should be doable in getTrivaFromIndex